RecoJets/JetAnalyzers/test/JetIdExample.py

```diff
@@ -42,7 +42,7 @@
    
 process = cms.Process("Ana")
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
-process.load("Configuration.StandardSequences.Geometry_cff")
+process.load("Configuration.StandardSequences.GeometryDB_cff")
 process.GlobalTag.globaltag = GLOBAL_TAG
```

Comment:
Hi @jeongeun `GLOBAL_TAG` is initialized with old GT; moreover, the input files used are from 2010.
Did you try to run it? Maybe we can directly remove this?

@cms-sw/jetmet-pog-l2 , what do you think?

Reply:
Hi @cms-sw/jetmet-pog-l2 , any thoughts about it?